packages/babel-preset-react-app/index.js

```diff
@@ -84,10 +84,13 @@ if (env === 'test') {
   module.exports = {
     presets: [
       // Latest stable ECMAScript features
-      [require.resolve('babel-preset-latest'), {
-        'es2015': {
-          modules: false
-        }
+      [require.resolve('babel-preset-env'), {
+        targets: {
+          // We support everything React supports
+          'ie': 9
+        },
+        // Disable polyfill transforms
+        useBuiltIns: false
```

Comment:
This is disabled by default but I explicitly set it false so it doesn't accidentally get enabled in a minor release.

Setting this to true requires npm 3 / node 6 and transforms `import "babel-polyfill";` into, e.g.:
```json
import "core-js/modules/es7.string.pad-start";
import "core-js/modules/es7.string.pad-end";
import "core-js/modules/web.timers";
import "core-js/modules/web.immediate";
import "core-js/modules/web.dom.iterable";
```